clients/admin-ui/src/home/HomeBanner.tsx

```diff
@@ -72,12 +58,15 @@ const HomeBanner: React.FC = () => {
           </>
         )}
       </Flex>
+      {/* Render the background image, using a min-width here to ensure there is
+      enough left margin to avoid colliding with the banner text above */}
```

Comment:
Thanks for leaving a comment about the `minWidth` below. I wasn't sure why it was there at first. Creative use of it there!